pkg/helpers/status.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helpers
```

Comment:
maybe we can name this package `status`? helpers is not too descriptive

Reply:
I went with `helpers` because I thought it might be useful to have a package for any future helper functions, but I'm fine with naming it more narrowly as `status`.

Reply:
@bobcatfish makes some arguments against helpers in https://bobcatfish.com/post/helpers/ - I think they are strong arguments, what do you think?

Reply:
Sounds fine by me! I'll rename to `status`.